cves/CVE-2019-11358.yml

```diff
@@ -49,7 +49,20 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The following Django vulnerability stems from the ability to use a property
```

Comment:
Consider changing "the following" to "this". It just seems more welcoming since you are already describing it.:)